tests/test_generics.py

```diff
@@ -825,7 +825,7 @@ class GenericSub(GenericBaseModel[int], Generic[U]):
 
 def test_generic_model_pickle(create_module):
     # Using create_module because pickle doesn't support
-    # objects with <locals> in their __qualname__  (e. g. defined in function)
+    # objects with <locals> in their __qualname__  (e.g. defined in function)
```

Comment:
I noticed that the majority of the codebase uses `e.g.` not `e. g.` (and pycharm was giving me a grammar warning for including the space lol). Because it affected one of the tests I just did a find-and-replace for the whole codebase, which is where this change is coming from.